File: Javascript/traversy-js-challenges-main/02-basic-challenges-2/05-find-missing-letter/find-missing-letter.js
function findMissingLetter(inputArr) {
    let realAlphabet ="abcdefghijklmnopqrstuvwxyz";
    let startingChar = inputArr[0];
    if (startingChar === startingChar.toUpperCase()) {
        realAlphabet = realAlphabet.toUpperCase();
    }
    let startIndex = realAlphabet.indexOf(startingChar);
    for (let i = startIndex; i < startIndex + inputArr.length; i++) {
        if (inputArr.indexOf(realAlphabet[i]) === -1) {
            return realAlphabet[i]
        }
    }
    return '';
}

module.exports = findMissingLetter;

File: Kotlin/AlgorithmsPractice/src/findMissing/FindMissingLetter.kt
package findMissing

class FindMissingLetter(
    private val inputArr: CharArray
) {
    var realAlphabet = "abcdefghijklmnopqrstuvwxyz"

    fun getResult(): Char {
        val startingChar = inputArr[0].toString();
        if (startingChar == startingChar.uppercase()) {
            realAlphabet = realAlphabet.uppercase();
        }
        val startIndex = realAlphabet.indexOf(startingChar);
        for (i in startIndex..startIndex + inputArr.size) {
            if (inputArr.indexOf(realAlphabet[i]) == -1) {
                return realAlphabet[i]
            }
        }
        return 0.toChar()
    }
}